td.vue/src/views/git/ThreatModelSelect.vue

```diff
@@ -5,6 +5,7 @@
         :emptyStateText="$t('threatmodelSelect.newThreatModel')"
         :onEmptyStateClick="newThreatModel">
             {{ $t('threatmodelSelect.select') }}
+            <!-- Fixme: The href should get the configured hostname from env -->
```

Comment:
the idea you suggested in your pr comments looks a good one to me, access this value from td.server via a new API which are listed in `td.server/src/config/routes.config.js`